Basic-Scripts/String Validators.py

```diff
@@ -0,0 +1,24 @@
+#This is the program to find out if the string  contains: alphanumeric characters, alphabetical characters, digits, lowercase and uppercase characters.
+if __name__ == '__main__':
```

Comment:
Make a separate function for your algo. In `main` only take the input from user.
Follow [PEP8 Guidelines](https://www.python.org/dev/peps/pep-0008/)

Reply:
Updated String Validators.py